README.md

```diff
@@ -1,5 +1,7 @@
-hexo-math [![Build Status](https://travis-ci.org/hexojs/hexo-math.svg?branch=master)](https://travis-ci.org/hexojs/hexo-math)
-===================
+# hexo-math
+
+[![Build Status](https://travis-ci.org/hexojs/hexo-math.svg?branch=master)](https://travis-ci.org/hexojs/hexo-math)
+[![NPM version](https://badge.fury.io/js/hexo-math.svg)](https://www.npmjs.com/package/hexo-math)
 
```

Comment:
IMO add the Appveyor badge.
`[![Build status](https://ci.appveyor.com/api/projects/status/9yjyl5u5omxsidg8/branch/master?svg=true)](https://ci.appveyor.com/project/hexojs/hexo-math/branch/master)`

Reply:
I prefer to use https://ci.appveyor.com/api/projects/status/github/hexojs/hexo-math?branch=master&svg=true